packages/firebase_ml/example/test_driver/firebase_model_manager.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of 'firebase_ml.dart';
+
+void firebaseModelManagerTest() {
+  group('$FirebaseModelManager', () {
+    final FirebaseModelManager modelManager = FirebaseModelManager.instance;
+
+    test('downloadModel', () async {
+      FirebaseCustomRemoteModel model =
+          FirebaseCustomRemoteModel('image_classification');
+      assert(model != null);
+
+      FirebaseModelDownloadConditions conditions =
+          FirebaseModelDownloadConditions(requireWifi: true);
+
+      await modelManager.download(model, conditions);
+      expect(model.modelHash, isNotNull);
+
+      var isModelDownloaded = await modelManager.isModelDownloaded(model);
+      expect(isModelDownloaded, isTrue);
+
+      var modelFile = await modelManager.getLatestModelFile(model);
+      expect(modelFile, isNotNull);
+      expect(modelFile.path.runtimeType, ''.runtimeType);
```

Comment:
Yes, but these are two separate checks.
If modelFile is null, it will fail on calling .path, something that I want to check with different expect.

Reply:
oh sorry I meant on line 27, can we do something like `expect(modelFile.path, 'abc')`? This will cover the runtimeType check as well

Reply:
Not really, because it is a real file that is downloaded, and, I am not sure how is path is generated. Previously it was abstracted away, and only now users can access the file with the latest getLatestFile API.

Reply:
I can be more thorough, however, on what it should contain, such as name of the model (at some point)

Reply:
Done